docs/src/components/Typography/TypeScale.js

```diff
@@ -57,21 +57,18 @@ const formatSizes = sizes => {
 
 const getClassName = cssModules(STYLES);
 
-const TypeScale = props => {
-  const { className, ...rest } = props;
-
+const TypeScale = () => {
   return (
     <div
       aria-label="Type scale information"
-      className={getClassName('bpk-docs-type-scale', className)}
-      {...rest}
+      className={getClassName('bpk-docs-type-scale')}
```

Comment:
Removed the overriding className and rest here as they weren't being used in the old implementation so added no value